uint8 servo_id
uint8 status
uint16 position
int16 speed
int16 load